drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -1522,8 +1522,9 @@ static int vidioc_s_fmt(struct bcm2835_codec_ctx *ctx, struct v4l2_format *f,
 	q_data->fmt = find_format(f, ctx->dev,
 				  f->type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE);
 	q_data->crop_width = f->fmt.pix_mp.width;
+	q_data->crop_height = f->fmt.pix_mp.height;
```

Comment:
This changes behaviour for the other roles.

This one does feel weird as to why it is required - crop_height should come through s_selection, so why is it needed here?

Reply:
Can you just check whether this and the change 2 lines later are actually still needed? Now we're calling s_selection on the correct buffer type they feel unnecessary.